runlike/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.6.3"
+__version__ = "0.6.4"
```

Comment:
Since this change adds new features I think it warrants a minor version bump to 0.7.0 actually. No worries, I'll take care of that when releasing.